### ISSUE TITLE
 Monero Website Meeting - Sun 22 September 2024, 19:00 UTC

### ISSUE THREAD

**Issue by monerobull** [monero-project/meta#1073]
Location: [Matrix](https://matrix.to/#/#monero-site:monero.social) | [Libera.chat, #monero-site](https://libera.chat)

Time: 19:00 UTC [Check in your timezone](https://www.timeanddate.com/worldclock/converter.html?iso=20211027T190000&p1=1440)

Main discussion topics:

    Greetings

    Reviewing community input 

    Any other business

    Confirm next meeting agenda

Please comment on GitHub in advance of the meeting if you would like to propose an agenda item.

**Comment by rottenwheel** [comment]
@plowsof please work your magic so logs are posted for website automatically, if possible. Meeting adjourned yesterday, logs should be available to be posted, issue closed and new one to be put up. 👍

**Comment by plowsof** [comment]
Logs 
> __< geonic >__ meeting in 0 minutes     

> __< c​hch3003:monero.social >__ 🔔🔔🔔     

> __< geonic >__ meeting in -3 minutes     

> __< geonic >__ alright...     

> __< geonic >__ 1. Greetings     

> __< geonic >__ hello     

> __< b​asses:matrix.org >__ hello     

> __< c​hch3003:monero.social >__ Hi     

> __< c​hch3003:monero.social >__ Is there an agenda?     

> __< b​asses:matrix.org >__ https://github.com/monero-project/meta/issues/1073     

> __< c​hch3003:monero.social >__ Ok, we are 3     

> __< c​hch3003:monero.social >__ ?     

> __< c​hch3003:monero.social >__ Ok, I have something to say     

> __< d​iego:cypherstack.com >__ hi     

> __< b​asses:matrix.org >__ what is even that agenda, very vague     

> __< d​iego:cypherstack.com >__ apologies for my tardiness     

> __< d​iego:cypherstack.com >__ I can run the meeting, if monerobull isn't here     

> __< c​hch3003:monero.social >__ Hey!     

> __< geonic >__ yay 4     

> __< geonic >__ go ahead diego as I'm afk     

> __< d​iego:cypherstack.com >__ Janaka, you said you had something to say. Go for it.     

> __< c​hch3003:monero.social >__ Ok     

> __< c​hch3003:monero.social >__ After months of contemplating how a new Monero website should look, it became clear to me that it should be in light mode. Dark mode, while visually appealing, doesn’t convey the level of professionalism required. Light mode is the way forward.      

> __< c​hch3003:monero.social >__ Also, I would not support both; as maintaining both options is too much work for too little benefit.     

> __< c​hch3003:monero.social >__ A few months ago, I began working on the Hammerman design because I was impressed by the overall design and wanted to contribute to the community, so I went ahead and implemented the design.     

> __< c​hch3003:monero.social >__ However, I’ve since changed my mind. I initially got excited about dark mode, but now I don't think this design should represent the official Monero website. Dark mode is a trap.     

> __< c​hch3003:monero.social >__ While we heard some voices in favor of dark mode, I think going light-mode only is aligned with a good portion of the community, as shown by this Reddit thread: https://www.reddit.com/r/Monero/comments/1felb8e/the_future_of_getmoneroorg_looking_for_feedback/     

> __< c​hch3003:monero.social >__ Going forward, I believe we should commission one or more designers to create a light mode theme that is sober, clean, and professional—similar to the current design. Once that's in place, we can carefully curate the content for the website.      

> __< c​hch3003:monero.social >__ Only after that should we consider the technical implementation, which is a separate matter entirely.     

> __< d​iego:cypherstack.com >__ Understood. Thank you Janaka     

> __< d​iego:cypherstack.com >__ does anyone in attendance have thoughts or comments on that?     

> __< geonic >__ it's clear from the community feedback we've received that light mode is preferred     

> __< d​iego:cypherstack.com >__ Another point of the community feedback that is largely consistent is that a redesign isn't needed at all.     

> __< geonic >__ and since we already have light mode, this leads us to a more fundamental question -- what is this redesign hoping to achieve?     

> __< geonic >__ yes     

> __< d​iego:cypherstack.com >__ I can provide an answer to this question, although I'm not necessarily 100% gung ho on the answers I'd be giving.     

> __< b​asses:matrix.org >__ I say support both dark and light theme     

> __< d​iego:cypherstack.com >__ I think one of the primary ones is moving to a new, modern framework would be beneficial. It would make things like translations and other stuff easier.     

> __< d​iego:cypherstack.com >__ And if we moved for the sake of framework, I think a lot of people thought it'd be a shame to not use the opportunity for a refresh.     

> __< m​onerobull:matrix.org >__ hello, sorry for being late to the meeting i announced myself     

> __< m​onerobull:matrix.org >__ we've got a bunch of feedback on both reddit and monero.town     

> __< m​onerobull:matrix.org >__ https://monero.town/post/4289749?scrollToComments=true     

> __< m​onerobull:matrix.org >__ https://www.reddit.com/r/Monero/comments/1felb8e/the_future_of_getmoneroorg_looking_for_feedback/     

> __< plowsof >__ 👋     

> __< c​hch3003:monero.social >__ In my opnion, the main benefit is to renew the tech stack. We can have a new design, but nothing fancy, light mode only. The tech stack should allow easy contribution and translation.     

> __< d​iego:cypherstack.com >__ Right, both of these links have people that are saying a refresh is probably not necessary.     

> __< m​onerobull:matrix.org >__ the three top comments on reddit are "keep it the same"     

> __< m​onerobull:matrix.org >__ https://matrix.monero.social/_matrix/media/v1/download/matrix.org/RmwnpwMbueNpERRfwQLzvcOp     

> __< d​iego:cypherstack.com >__ Keep in mind, the same was said when I first redesigned the site by a ton of people.     

> __< d​iego:cypherstack.com >__ With the design before mine     

> __< d​iego:cypherstack.com >__ About the design before mine     

> __< d​iego:cypherstack.com >__ Janaka what are your thoughts of remaking the current site with the new tech stack?     

> __< d​iego:cypherstack.com >__ obviously we use the opportunity to update content and images     

> __< d​iego:cypherstack.com >__ but keep the general look and feel, the blocks, fonts, etc.     

> __< o​frnxmr:monero.social >__ Jekyll is a problem, which is why we were looking at hugo iirc     

> __< c​hch3003:monero.social >__ Yes we can keep the design as is also. Just that it's nice to change a little sometimes     

> __< m​onerobull:matrix.org >__ it is a static site, how bad can it really be?     

> __< o​frnxmr:monero.social >__ Jekyll is EOL or smthn     

> __< m​onerobull:matrix.org >__ it's still static     

> __< m​onerobull:matrix.org >__ this isn't wordpress     

> __< o​frnxmr:monero.social >__ https://github.com/monero-project/monero-site/issues/2140     

> __< o​frnxmr:monero.social >__ ```     

> __< o​frnxmr:monero.social >__ there is a problem that we have to deal with: jekyll-multiple-languages-plugin which we are using to make the website multilingual has been in maintainance mode for years. It was always a bit of a problem, but now things are getting worse, as incompatibilities are starting to pop up which force us to lock some specific versions in the dependencies.     

> __< o​frnxmr:monero.social >__ This is a problem that will only get worse and that could only be solved by somebody taking over the plugin and maintain it again. This hasn't happened yet and i feel it won't ever happen. The result could be that we will find ourselves unable to update to newer Jekyll versions or we could find ourselves stuck in a dependency hell.     

> __< o​frnxmr:monero.social >__ ```     

> __< m​onerobull:matrix.org >__ we can also just remove moneropedia     

> __< d​iego:cypherstack.com >__ Brand new videos are being made which can replace the old ones     

> __< d​iego:cypherstack.com >__ there's like ten actual illustrations on the site. All of them can be replaced. We can ask andres to do them all for a consistent feel     

> __< c​hch3003:monero.social >__ We can use Hugo if you want but personally, I am much more familiar with Typescript than Go.     

> __< c​hch3003:monero.social >__ Someone else can implement the website though.     

> __< d​iego:cypherstack.com >__ The old illustrations were made by me to match the video at the bottom of this page?     

> __< d​iego:cypherstack.com >__ https://www.getmonero.org/get-started/what-is-monero/     

> __< o​frnxmr:monero.social >__ Im not suggesting hugo     

> __< plowsof >__ i fixed the jekyll language thing.. and some other small bugs preventing update to the latest ruby but uhm yeah some issues remain with po4a .. small bug in how we have key lists in our file ... what im saying is the current tech stack is fragile      

> __< o​frnxmr:monero.social >__ i'm making note of jekyll issue     

> __< plowsof >__ that plugin is abandonware* i merely made it run      

> __< c​hch3003:monero.social >__ If we all agree that we should keep it light mode, more or less as it is now, we are making a big step forward     

> __< d​iego:cypherstack.com >__ I'm fine with all of that, sure.     

> __< plowsof >__ its here, i have repressed the memories https://github.com/monero-project/monero-site/pull/2237     

> __< d​iego:cypherstack.com >__ Thoughts on asking Andres to do the illustrations?     

> __< c​hch3003:monero.social >__ I have nothing against it     

> __< d​iego:cypherstack.com >__ not a 1-to-1 replacement of current images. First step would be to do adjustments on content.     

> __< d​iego:cypherstack.com >__ After that's done we can decide what images are needed and can ask him     

> __< d​iego:cypherstack.com >__ I ask because his style is unique and identifiable. And some people may want things a little simpler     

> __< plowsof >__ removing moneropedia is more realistic now that there is a docs workgroup      

> __< b​asses:matrix.org >__ if we are going to add new vids from vost, then dark theme makes sense 🤷     

> __< d​iego:cypherstack.com >__ For those who don't know he did all the imagery for tailsOS     

> __< d​iego:cypherstack.com >__ https://tails.net/     

> __< d​iego:cypherstack.com >__ and he's also the creator of the iconic Monero bricks community image at the bottom of the homepage     

> __< plowsof >__ in vostos latest ccs , light and dark mode themed would be produced if funded https://ccs.getmonero.org/proposals/VOSTOEMISIO-FCMP-Animated-Explainer.html     

> __< o​frnxmr:monero.social >__ we're ready to redirect rpc docs now     

> __< m​onerobull:matrix.org >__ id bet money this is the same person that did the ringsig image     

> __< c​hch3003:monero.social >__ Tails is light mode only btw...     

> __< d​iego:cypherstack.com >__ as is tor     

> __< d​iego:cypherstack.com >__ https://www.torproject.org/     

> __< c​hch3003:monero.social >__ I mean, they got it.     

> __< c​hch3003:monero.social >__ Or Bitcoin     

> __< c​hch3003:monero.social >__ Lot things     

> __< d​iego:cypherstack.com >__ I've always been partial to light mode myself, yeah. Hence current live design.     

> __< c​hch3003:monero.social >__ Lot of things     

> __< d​iego:cypherstack.com >__ and, if I may be blunt, since this meeting was broadcast ahead of time, by quite a ways. I'm inclined to care more about the opinions that are here     

> __< geonic >__ great to see a change of heart and a sudden embrace of light mode     

> __< geonic >__ ofrnxmr: where is that quote from?     

> __< b​asses:matrix.org >__ u are sitting at night with lights off, you open your fav light only theme website, flashbanged     

> __< o​frnxmr:xmr.mx >__ https://matrix.monero.social/_matrix/media/v1/download/xmr.mx/jdEIfkvHaygZnxqnWxGjAdxm     

> __< c​hch3003:monero.social >__ rando: Just don't watch screen at night     

> __< c​hch3003:monero.social >__ rando: Just don't watch screens at night     

> __< o​frnxmr:monero.social >__ Geonic - from the github i linked     

> __< b​asses:matrix.org >__ I'm talking practically     

> __< c​hch3003:monero.social >__ Or turn on the light     

> __< d​iego:cypherstack.com >__ I think if some people want to make dark mode for the site on the new framework, they should be able to do so, obviously. It's a community project, after all.     

> __< o​frnxmr:monero.social >__ https://github.com/monero-project/monero-site/issues/2140 geonic     

> __< geonic >__ ah tx     

> __< d​iego:cypherstack.com >__ Janaka is just not going to add or upkeep it     

> __< b​asses:matrix.org >__ yes, obviously there are also people that want dark theme, so why not support both?     

> __< b​asses:matrix.org >__ give people the option to choose what they want?     

> __< d​iego:cypherstack.com >__ And having both will be much easier on a new framework     

> __< c​hch3003:monero.social >__ Yes, I think that sometimes users are wrong     

> __< b​asses:matrix.org >__ including you     

> __< b​asses:matrix.org >__ this is a community project     

> __< c​hch3003:monero.social >__ Yes yes haha It happens     

> __< o​frnxmr:monero.social >__ It takes 0.1sec for my browser to do it. 0.1 sec for docs.getmonero.org     

> __< b​asses:matrix.org >__ stop flexing docs for a min lol     

> __< d​iego:cypherstack.com >__ well, janaka I can understand you saying you will not implement or upkeep it. But if you're saying it shouldn't be added by anyone else, I would disagree strongly.     

> __< o​frnxmr:monero.social >__ Docs has light + dark + autodetect     

> __< d​iego:cypherstack.com >__ white mode as default, I agree btw     

> __< m​onerobull:matrix.org >__ agreed     

> __< o​frnxmr:monero.social >__ No     

> __< o​frnxmr:monero.social >__ Docs is an extension of site and should flow nicely     

> __< o​frnxmr:monero.social >__ Shouldnt get flashbanged when clicking through     

> __< c​hch3003:monero.social >__ But, my opinion today is that light mode only is better for maintainability. Supporting dual theme without JS means compiling for both, and generating different urls, it's a mess     

> __< c​hch3003:monero.social >__ Lot of work for very little benefit     

> __< d​iego:cypherstack.com >__ then perhaps we make light mode only available to those with JS turned on?     

> __< d​iego:cypherstack.com >__ The site "degrades" nicely without JS in that literally all content is still available, just only in light mode :P     

> __< m​onerobull:matrix.org >__ if we do darkmode by default that isnt great     

> __< plowsof >__ there is a simple way of changing style sheets of course (in use circa 1999)     

> __< m​onerobull:matrix.org >__ this is mostly psychology     

> __< c​hch3003:monero.social >__ YES     

> __< m​onerobull:matrix.org >__ darkmode by default we might as well get sponsorships by DNMs     

> __< geonic >__ tbh we’re lacking translations not because of a technical hurdle but because of an incoherent policy where we paid a “translations coordinator”  instead of the people doing the work (QA issues aside)     

> __< b​asses:matrix.org >__ https://developer.mozilla.org/en-US/docs/Web/CSS/@media/prefers-color-scheme     

> __< o​frnxmr:xmr.mx >__ We are sponsored by majestic and were sponsored by forked.     

> __< o​frnxmr:xmr.mx >__ "<g​eonic> tbh we’re lacking translations not because of a technical hurdle but because of an incoherent policy where we paid a “translations coordinator”  instead of the people doing the work (QA issues aside)"      

> __< o​frnxmr:xmr.mx >__ wrong..     

> __< m​onerobull:matrix.org >__ and monerokon by exch     

> __< m​onerobull:matrix.org >__ so what     

> __< m​onerobull:matrix.org >__ getmonero.org is still looking friendly :D     

> __< b​asses:matrix.org >__ literally exists     

> __< b​asses:matrix.org >__ bruh     

> __< c​hch3003:monero.social >__ It's a trap to think Monero == privacy == darknet == dark mode     

> __< c​hch3003:monero.social >__ No! Any serious projects use light mode only.     

> __< c​hch3003:monero.social >__ Dark mode is suited for specific apps, or ICO websites, but not for serious projects     

> __< o​frnxmr:xmr.mx >__ Generalfubd has thousands of neros deposited by unknown whale. Pretty sure were sponsored why who knows     

> __< o​frnxmr:xmr.mx >__ By* who knows     

> __< b​asses:matrix.org >__ duckduckgo supports dark theme     

> __< m​onerobull:matrix.org >__ agreed     

> __< plowsof >__ are you suggesting wownero isnt a serious project because it has a darkish theme site https://wownero.org/     

> __< m​onerobull:matrix.org >__ darkmode is to raise VC funding     

> __< b​asses:matrix.org >__ actually it is a trend for privacy products to support dark theme     

> __< c​hch3003:monero.social >__ rando: Ok but also take into considerence the maintenance cost. It is not worth it     

> __< c​hch3003:monero.social >__ rando: Ok but also take into consideration the maintenance cost. It is not worth it     

> __< b​asses:matrix.org >__ how so? enlighten me     

> __< m​onerobull:matrix.org >__ https://www.expressvpn.com/     

> __< d​iego:cypherstack.com >__ Guys, this is all a bit silly. Having an non-default OPTION for dark mode doesn't make Monero look like a less serious project. It makes it look like a more accessible one.     

> __< o​frnxmr:xmr.mx >__ Even repo.getmonero.org has dark theme     

> __< m​onerobull:matrix.org >__ does this look like a shady site to you?     

> __< d​iego:cypherstack.com >__ "Dark mode" doesn't have to mean OLED black and hacker green text     

> __< o​frnxmr:xmr.mx >__ We even have qr codes in dark and light.     

> __< b​asses:matrix.org >__ doesn't load without JS, not a serious project     

> __< o​frnxmr:xmr.mx >__ I thought we agreed to use the hacker/matrix scrolling txt?     

> __< d​iego:cypherstack.com >__ It seems pretty obvious to me that what should happen going forward is that Janaka will remake the current site (with content changes) in a new framework. Janaka will be doing light mode only. After the site is done, people that are NOT Janaka can utilize their skills to give the website a dark theme as well.     

> __< o​frnxmr:xmr.mx >__ Scroll down. It loads the footer 💯💯     

> __< u​ntraceable:monero.social >__ How about a dark theme that isn't black, but dark gray like https://www.ivpn.net/en/     

> __< b​asses:matrix.org >__ like guys cmon lmao, you are making supporting dark theme a crime, ignoring all accessibility and user's choice     

> __< c​hch3003:monero.social >__ We don't want two themes because it is not necessary.     

> __< geonic >__ who’s we lol     

> __< b​asses:matrix.org >__ perfecto     

> __< m​onerobull:matrix.org >__ no offense but what a bad website. wall of text with "hacker" font     

> __< u​ntraceable:monero.social >__ I think it should definitely be an option, but not the default     

> __< o​frnxmr:xmr.mx >__ Repo.getmonero and docs.getmonero     

> __< m​onerobull:matrix.org >__ i like the expressvpn one a lot more     

> __< o​frnxmr:xmr.mx >__ Penpot.getmonero.org is also not "light"     

> __< 3​21bob321:monero.social >__ Ban     

> __< u​ntraceable:monero.social >__ I'm only talking about the color, not design     

> __< b​asses:matrix.org >__ says the guy that like cypherpunk theme websites     

> __< m​onerobull:matrix.org >__ i like retrowave neon stuff     

> __< m​onerobull:matrix.org >__ avax website: terrible     

> __< m​onerobull:matrix.org >__ actually depressing     

> __< b​asses:matrix.org >__ everyone have their taste     

> __< b​asses:matrix.org >__ but we are talking about accessbility here     

> __< c​hch3003:monero.social >__ It has a dev/design cost. And we lose the brand.      

> __< c​hch3003:monero.social >__ A website is not an app.      

> __< c​hch3003:monero.social >__ An application can have dual mode.     

> __< c​hch3003:monero.social >__ A website needs an identity, and just one.     

> __< d​iego:cypherstack.com >__ We're starting to derail a bit here.     

> __< 3​21bob321:monero.social >__ Still no consensus after 3 meetings …lol     

> __< b​asses:matrix.org >__ so google lost their brand for supporting dark them for most if not all their services?     

> __< o​frnxmr:xmr.mx >__ github has dark theme     

> __< geonic >__ Agreed. Most people are perfectly happy with the design. If the translations are the reason we’re considering switching platforms let’s talk about that.     

> __< b​asses:matrix.org >__ I give up lol     

> __< c​hch3003:monero.social >__ But this is more like apps     

> __< u​ntraceable:monero.social >__ How about a vote to settle it     

> __< o​frnxmr:xmr.mx >__ This is 4, iirc     

> __< d​iego:cypherstack.com >__ it'd be funny if it wasn't so frustrating :P     

> __< c​hch3003:monero.social >__ Well, anyone who disagree with me. It's fine, but take the time to really think about it please.     

> __< b​asses:matrix.org >__ vscode, literally all 90% of programmers use, because it helps programmers not going blind by staring at sun looking screen 12h avg     

> __< geonic >__ no need to vote on something when community consensus is pretty clear     

> __< d​iego:cypherstack.com >__ geonic what's your take on what the consensus is?     

> __< o​frnxmr:xmr.mx >__ Github, google, bing, ddg, repo.getmonero, docs.getmonero, penpot.getmonero, all have dark choice     

> __< geonic >__ unless we’re somehow more special than the masses     

> __< b​asses:matrix.org >__ no one is maintaining Weblate, another problem for another dimension     

> __< geonic >__ that people like the current design. don’t fix what’s not broken, etc     

> __< o​frnxmr:xmr.mx >__ and devices like phones even _auto switch_ depending on time of day     

> __< c​hch3003:monero.social >__ Yes, those are APPLICATIONS     

> __< d​iego:cypherstack.com >__ (asking about the dark thing btw)     

> __< c​hch3003:monero.social >__ NOT WEBSITES     

> __< 3​21bob321:monero.social >__ Is this about having a dark theme     

> __< o​frnxmr:xmr.mx >__ Those are websites bro     

> __< o​frnxmr:xmr.mx >__ Github.com is a website and an app (both have dark theme)     

> __< d​iego:cypherstack.com >__ alright everyone. The dark theme is a bit of a moot point at the moment anyways. Once again, it can be added rather easily, after the fact, since we'd be on a new, more modern framework.     

> __< b​asses:matrix.org >__ nah war lol     

> __< c​hch3003:monero.social >__ I mean, github.com is more an app than a website     

> __< 3​21bob321:monero.social >__ Is this not a small thing to argue about. Dark and light mode welcome to the 21st century     

> __< o​frnxmr:xmr.mx >__ https://matrix.monero.social/_matrix/media/v1/download/xmr.mx/gXlnVGROZbrTikfEKWNKrrKx     

> __< b​asses:matrix.org >__ wikis are apps now?     

> __< d​iego:cypherstack.com >__ What is clear from this meeting (to me) is the following:     

> __< d​iego:cypherstack.com >__ 1. Keep current design     

> __< d​iego:cypherstack.com >__ 2. Update to new framework     

> __< d​iego:cypherstack.com >__ 3. Tweak existing content, illustrations, and replace videos     

> __< 3​21bob321:monero.social >__ I will send pagers!     

> __< b​asses:matrix.org >__ because wikipedia started supproting dark theme recently, also arch linux wiki     

> __< o​frnxmr:xmr.mx >__ Update content and strings     

> __< geonic >__ 2. was never discussed in any detail     

> __< d​iego:cypherstack.com >__ 4. Janaka can do the above, but will only be doing light theme     

> __< b​asses:matrix.org >__ can we talk about the actual website for real now?     

> __< geonic >__ diego making up action items again     

> __< u​ntraceable:monero.social >__ Wait, I arrived a bit late. There's still going to have a redesign, right? Just a light default theme instead of dark as proposed?     

> __< 3​21bob321:monero.social >__ To me janka has the only demo site currently     

> __< b​asses:matrix.org >__ how many videos will be replaced, all?     

> __< m​onerobull:matrix.org >__ Community seems to be against it     

> __< d​iego:cypherstack.com >__ wait, geonic, but you just said that the consensus as you saw it was that a new design wasn't needed?     

> __< u​ntraceable:monero.social >__ Ah whaaat I was excited for a redesign.     

> __< 3​21bob321:monero.social >__ Can diego convert his design into demo site?     

> __< geonic >__ right. no discussion has been had on the framework though     

> __< d​iego:cypherstack.com >__ geonic, we have discussed the framework for many, many meetings     

> __< o​frnxmr:xmr.mx >__ astro for same design revamp?     

> __< d​iego:cypherstack.com >__ and if you can please disagree with me in a way that is not the most obnoxious possible, that would be appreciated     

> __< geonic >__ are we back to “consensus was already achieved” lol     

> __< c​hch3003:monero.social >__ We will refresh the tech stack, fix translation, light mode only, stop crying it's ok, make Monero great again!!     

> __< b​asses:matrix.org >__ and they gave bad reasonings, dark = bad = crime.  Worst conclusion     

> __< o​frnxmr:xmr.mx >__ Racists     

> __< geonic >__ framework is not even on the agenda for today     

> __< o​frnxmr:xmr.mx >__ We should make it bright pink     

> __< o​frnxmr:xmr.mx >__ Like cake wallet     

> __< geonic >__ did we decide Hugo is best because ErC said so?     

> __< d​iego:cypherstack.com >__ it's not just me that seems to think so geonic? janaka and ofrn seem to think so too?>     

> __< m​onerobull:matrix.org >__ No the main criticism was if it ain't broke don't fix it     

> __< b​asses:matrix.org >__ yeah this not kindergarden     

> __< geonic >__ Janaka is looking for a job so he’s not exactly unbiased here     

> __< d​iego:cypherstack.com >__ no geonic. We would be using Astro. Once again, previous meetings.     

> __< o​frnxmr:xmr.mx >__ Geonic - no, i just pointed out that jekyll was not a good option     

> __< 3​21bob321:monero.social >__ You know sometimes a dictator is good. Shit gets done…..     

> __< b​asses:matrix.org >__ it is actually broken, they are just seeing the surface     

> __< d​iego:cypherstack.com >__ I don't know anyone who actually works on the site who really wants to see Jekyll stay?     

> __< 3​21bob321:monero.social >__ Plowsof     

> __< o​frnxmr:xmr.mx >__ No     

> __< 3​21bob321:monero.social >__ Anything but rust     

> __< o​frnxmr:xmr.mx >__ I mean, i think were all ok with dumping jekyll     

> __< d​iego:cypherstack.com >__ someone else must take the reins there sir     

> __< d​iego:cypherstack.com >__ no, no. geonic wasn't there for the many conversations where it happened, so it doesn't count     

> __< c​hch3003:monero.social >__ I will if you don't     

> __< d​iego:cypherstack.com >__ and any attempt to point back to those conversations is railroading     

> __< b​asses:matrix.org >__ no     

> __< 3​21bob321:monero.social >__ But i dont get paid to whip people     

> __< geonic >__ There’s only one comment on ErC’s github issue about the framework. This was never the long-standing issue you’re making it out to be     

> __< d​iego:cypherstack.com >__ I can't or geonic will bitch, so please do     

> __< b​asses:matrix.org >__ agenda is vague, and technically doesn't exist     

> __< geonic >__ I get that you’re looking for something to do tho     

> __< d​iego:cypherstack.com >__ we have had IRC meetings     

> __< m​onerobull:matrix.org >__ ok guys we are coming up on the one hour mark     

> __< o​frnxmr:xmr.mx >__ Well then i say light and dark, implement diego's design with some tweaks and keep old site around at old.getmonero.org     

> __< b​asses:matrix.org >__ +1     

> __< 3​21bob321:monero.social >__ I have a homelab there is always something else to do.     

> __< m​onerobull:matrix.org >__ i think we can all agree that a default light + optional dark mode is the goal     

> __< o​frnxmr:xmr.mx >__ +1 with rando and dan     

> __< 3​21bob321:monero.social >__ Consensus meet     

> __< c​hch3003:monero.social >__ So ok, Diego Salazar you can work on a light mode design close to the current one, just make it even better. Then we will talk about implementation     

> __< o​frnxmr:xmr.mx >__ untraceable     

> __< d​iego:cypherstack.com >__ uh wait. Sorry if I'm being an idiot. Are you talking about my most recent design that I put forth a couple weeks back?>     

> __< 3​21bob321:monero.social >__ Is this how pow works?     

> __< d​iego:cypherstack.com >__ cuz if so, that one is out     

> __< o​frnxmr:xmr.mx >__ Yes     

> __< m​onerobull:matrix.org >__ early bird gets the 🪱     

> __< o​frnxmr:xmr.mx >__ Out where     

> __< b​asses:matrix.org >__ not it in hurry, just need to make sure at least we are one step forward or there is a progress     

> __< d​iego:cypherstack.com >__ Once again, sorry for being a moron. But I'm so confused. Why would I do that? I thought the consensus here was that we stick close to the current, live design? lol     

> __< geonic >__ ofrn: community voted no on the proposed designs     

> __< plowsof >__ i have worked with jekyll, i couldnt tell you what it is.. sometimes i use/see liquid scripting.. ruby is a PITA to set up a local dev environment.. as a contributor im not too worried about the backend getting in the way      

> __< plowsof >__ a new backend getting in the way*     

> __< geonic >__ thanks plowsof     

> __< 3​21bob321:monero.social >__ I8n plowsof?     

> __< o​frnxmr:xmr.mx >__ Reddit voted no with fake bot votes     

> __< o​frnxmr:xmr.mx >__ And reddit blocks tor, as does monero,town     

> __< 3​21bob321:monero.social >__ Majestic?     

> __< geonic >__ I’m against paid contractors creating busywork for themselves without clearly articulated reasons     

> __< c​hch3003:monero.social >__ Hmm because It would be nice to have a Figma to implement it..     

> __< geonic >__ if you want to change the backend, comment on the github issue that’s been open for a year     

> __< plowsof >__ translations , even with a new fancy end still rely on >weblate< (the thing that we are unable to keep online or gets neglected and falls out of sync with whats on site)      

> __< b​asses:matrix.org >__ anyway, for at least the homepage dark theme + new vids will look a bit like a redesign     

> __< m​onerobull:matrix.org >__ dont act like a significant number of people were oppressed. reddit audience are the same exact normies that getmonero.org is supposed to reach     

> __< o​frnxmr:xmr.mx >__ plowsof, and all new content     

> __< b​asses:matrix.org >__ if we are not going to do any actual redesings and just switching frameworks     

> __< d​iego:cypherstack.com >__ No like I'm happy to make a figma design of the current, live, design that we can edit with content changes     

> __< m​onerobull:matrix.org >__ it doesnt need to target hardcore tor & monero users     

> __< b​asses:matrix.org >__ repost on Dread? lol     

> __< o​frnxmr:xmr.mx >__ More than significant. Youre biased cuz u block everybody on tor     

> __< geonic >__ basses: the new vids that are currently available are not up to standards. plus dark mode only.     

> __< d​iego:cypherstack.com >__ the confusion came because it seemed they were asking me to make a light mode of my design I pitched a couple of weeks ago     

> __< o​frnxmr:xmr.mx >__ Worse than reddit     

> __< 3​21bob321:monero.social >__ https://github.com/monero-project/monero-site/discussions/2372     

> __< o​frnxmr:xmr.mx >__ Cant even view .town     

> __< m​onerobull:matrix.org >__ yes i hate tor users with a burning passion /s     

> __< plowsof >__ new content overwrites the current stuff, and we await for the team of translators to adjust.. review.. and push 💪 or things dont get updated     

> __< o​frnxmr:xmr.mx >__ Clearly     

> __< c​hch3003:monero.social >__ Diego Salazar: Yes that's fine.     

> __< b​asses:matrix.org >__ up to standard?     

> __< m​onerobull:matrix.org >__ if you pay for the CDN ill take any ddos head on ofrn     

> __< geonic >__ i.e. they’re worse than the current ones     

> __< 3​21bob321:monero.social >__ We have standards??     

> __< plowsof >__ the current backend doesnt get in the way either*      

> __< d​iego:cypherstack.com >__ alright. I'll work on that then.     

> __< plowsof >__ just figure it out and forget the pain      

> __< d​iego:cypherstack.com >__ I'll do it in PenPot also.     

> __< o​frnxmr:xmr.mx >__ Who needs a cdn when 700tb is like 100$?     

> __< d​iego:cypherstack.com >__ so it will be easy for people to tweak content stuff     

> __< m​onerobull:matrix.org >__ do you think monero.town generates $100     

> __< 3​21bob321:monero.social >__ Kuno     

> __< o​frnxmr:xmr.mx >__ And servers.guru offered to cover the bill     

> __< 3​21bob321:monero.social >__ Sponsorship     

> __< plowsof >__ monero town was flooded with resources totalling over 100kusd , ofcourse it generates more than $100     

> __< geonic >__ how hard is it to add dark mode in Jekyll?     

> __< plowsof >__ adding dark mode would be  a"change style" button      

> __< plowsof >__ click , and the url remains the same     

> __< m​onerobull:matrix.org >__ monerica was flooded with what would make it the second most popular website after google.com lol     

> __< 3​21bob321:monero.social >__ Sounds clunky     

> __< geonic >__ do that then?     

> __< b​asses:matrix.org >__ monerobull you can turn off CF captcha when there's no attacks     

> __< plowsof >__ maybe janaka is talking about a more compreensive redeisng for dark/light?     

> __< c​hch3003:monero.social >__ Diego Salazar: I didn't review each piece of text carefully. But when I worked on the demo I noticed lot of content that I would like to change. But I don't have everything in mind now.     

> __< 3​21bob321:monero.social >__ Neva     

> __< m​onerobull:matrix.org >__ rando but the attack is constant     

> __< o​frnxmr:xmr.mx >__ Its not captcha, its straight up blocked     

> __< m​onerobull:matrix.org >__ correct     

> __< m​onerobull:matrix.org >__ known tor exit nodes are blocked     

> __< 3​21bob321:monero.social >__ Ray id?     

> __< m​onerobull:matrix.org >__ because that is where the ddos is coming from     

> __< b​asses:matrix.org >__ call it bitcoin.town?     

> __< d​iego:cypherstack.com >__ Ok. I can't summarize what consensus is here necessarily. But I will say my personal, current, action plan based off of what I take from this meeting.     

> __< b​asses:matrix.org >__ thought Tor users like Monero     

> __< d​iego:cypherstack.com >__ I will be making a Figma / PenPot file of the CURRENT, LIVE site. Really just a 1 to 1 recreation.     

> __< d​iego:cypherstack.com >__ At which point, we as a community can play around with content and illustration changes.     

> __< d​iego:cypherstack.com >__ in an easy and non-committal way     

> __< geonic >__ diego: that was my proposals to you at the last meeting     

> __< geonic >__ tx     

> __< 3​21bob321:monero.social >__ Only crimes use it     

> __< c​hch3003:monero.social >__ Diego Salazar: Yes perfect     

> __< geonic >__ and again, whoever wants to change the backend, please articulate your reasons on Github https://github.com/monero-project/monero-site/issues/2140     

> __< b​asses:matrix.org >__ so can someone summarize on what decesions were made so we wont discuss it again in the next meeting?     

> __< d​iego:cypherstack.com >__ the general plan moving forward is that this site will be redone in Astro     

> __< o​frnxmr:xmr.mx >__ Id like to see what your design prospect looks like in light mode     

> __< d​iego:cypherstack.com >__ instead of Jekyll     

> __< geonic >__ it’s not a general plan Diego     

> __< d​iego:cypherstack.com >__ it is geonic     

> __< geonic >__ cool it     

> __< d​iego:cypherstack.com >__ no u     

> __< c​hch3003:monero.social >__ A nice Figma, properly organized, that the community can review.     

> __< o​frnxmr:xmr.mx >__ Plowsof, thought on moving to astro?     

> __< b​asses:matrix.org >__ decisions*     

> __< m​onerobull:matrix.org >__ ok guys please leave now     

> __< b​asses:matrix.org >__ ???     

> __< geonic >__ monerobull: say “meeting adjourned”     


Automated by [this](https://github.com/plowsof/post-libera-meeting-logs)
